### PR TITLE
Add iterative version and a start of the recursicve

### PR DESCRIPTION
**Pull request by fab-cloud9**
To solve the problem of showing a certain Fibonacci number a first solution using an iterative method is implemplemented. 

Also starrted a recursive version.

### CODE REVIEW

**Review by maxahlberg — APPROVED** [review]
:tada:

File: Master_File_Fibonacci.py [hunk 1]
```diff
@@ -1 +1,83 @@
...
+    fibbo_old_1 = 1  # 2nd initial latest old value of stored Fibonacci Number
+
+
+    if nn == 1:
+        return fibbo_old_2
+    elif nn == 2:
+        return fibbo_old_1
+    else:
+        return getFibboNumb(nn-1, fibbo_rec + fibbo_old_1 + fibbo_old_2)
+        fibbo_old_2 = fibbo_old_1
```
Comment (maxahlberg) [inline Master_File_Fibonacci.py]: I dont like the name `fibbo` use `fibo` instead

**Review by maxahlberg — CHANGES_REQUESTED** [review]

File: Master_File_Fibonacci.py (hunk 1 above)
Comment (fab-cloud9) [reply Master_File_Fibonacci.py]: `hej`

**Review by maxahlberg — APPROVED** [review]
:100:

**Pull request merged**